README.md

```diff
@@ -0,0 +1,27 @@
+# Unofficial PokeMMO flatpak
+
+<img src="https://pokemmo.com/build/images/pokemmo.cf970ad3.png">
+
+Welcome to a new era of online monster battles.
```

Comment:
no user will ever read this.

Reply:
It is important only for the wayland support section. Should i remove it?